### PR TITLE
Bug 1380563, updated Enabling Features in Technology Preview section

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1380563

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape PTAL

**Comment by ahardin-rh** [comment]
@bfallonf peer review please :bowing_woman:

**Comment by bfallonf** [comment]
@ahardin-rh LGTM :star2:

**Comment by ahardin-rh** [comment]
@jwforres PTAL. Thanks!

File: install_config/web_console_customization.adoc [hunk 1]
```diff
@@ -376,6 +376,41 @@ assetConfig:
 ----
 ====
 
+. As a cluster administrator, run the following command to give users the
```
Comment (jwforres) [inline install_config/web_console_customization.adoc]: this is really specific to jenkins pipelines, this wont be true of all tech preview features

**Review by jwforres — CHANGES_REQUESTED** [review]

File: install_config/web_console_customization.adoc (hunk 1 above)
Comment (jwforres) [inline install_config/web_console_customization.adoc]: @bparees are there docs about this in the build doc already?

Comment (bparees) [inline install_config/web_console_customization.adoc]: permission to use pipeline strategies should be on by default already, not sure where this is coming from?

i think all strategies are on by default, though we've been discussing turning off custom by default.

Comment (ahardin-rh) [inline install_config/web_console_customization.adoc]: @jwforres @bparees This was sourced from https://github.com/nickschuetz/ocp-3.3-pipelines/blob/master/README.md. However, it is noted that the steps arose from a possible edge case during an upgrade from 3.2 to 3.3, so this content may not be valid.

Comment (bparees) [inline install_config/web_console_customization.adoc]: yeah, i'm pretty sure that guy just didn't run reconcile roles which is a standard part of the upgrade process.

Comment (bparees) [inline install_config/web_console_customization.adoc]: so yes i'm pretty sure this should be closed.

**Pull request closed without merging**